src/libs/ReceiptUtils.js

```diff
@@ -0,0 +1,28 @@
+import lodashGet from 'lodash/get';
+import _ from 'underscore';
+import * as FileUtils from './fileDownload/FileUtils';
+import CONST from '../CONST';
+import Receipt from './actions/Receipt';
+import * as Localize from './Localize';
+
+const validateReceipt = (file) => {
+    const {fileExtension} = FileUtils.splitExtensionFromFileName(lodashGet(file, 'name', ''));
+    if (_.contains(CONST.API_ATTACHMENT_VALIDATIONS.UNALLOWED_EXTENSIONS, fileExtension.toLowerCase())) {
```

Comment:
@AndrewGable Hi! While migrating this file to TS I noticed that `CONST.API_ATTACHMENT_VALIDATIONS.UNALLOWED_EXTENSIONS` is used here, but it's not defined in `CONST.ts`. Could you provide a list of unallowed extensions so I can add it to the migration PR?